src/test/java/com/google/api/codegen/testdata/discoveries/go/go_adexchangebuyer.v1.4.json.baseline

```diff
@@ -141,10 +140,9 @@ func main() {
     log.Fatal(err)
   }
 
-  // TODO: Change placeholders below to desired parameter values for the `Patch` call:
-
   // The account id
   id := int64(0)
+  // TODO: Change placeholder value above.
```

Comment:
This does not work in general. Some comments associated with certain placeholder types (e.g., dates/times) cannot be pleasantly combined with that formatting. I think the syntax highlighting on API Explorer will make this at least as visually clear, and it's even easier to edit.